packages/signature-controller/src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const EthMethod = {
```

Comment:
Minor, this feels more like an enum as it's a fixed set of values to identify a type of something.

Which would also mean it could go in `types.ts`.

But we can do this later since it's not exported.

Reply:
I tried that but it is giving build issues somehow.